File: program/program/check-vowel/CheckVowel.ts
function checkVowels(str: string) {
    let isVowel = '';
    if (str == 'a' || str == 'e' || str == 'i' || str == 'o' || str == 'u' || str == 'A' || str == 'E' || str == 'I' || str == 'O' || str == 'U') {
        isVowel = 'Vowel'
    }
    else {
        isVowel = 'Is not a Vowel'
    }
    return isVowel;
}

console.log(checkVowels("U"));

File: program/program/check-vowel/CheckVowel.rb
#Author: suryapoojary0
#date: "13-10-2022"                             
puts "Input a letter in uppercase or smallcase" 
crux = ["a", "e" , "i", "o", "u", "A", "E", "I", "O", "U"]

num = gets.strip

if crux.include?(num) then
        puts "Vowel"
else                                                    puts "Not a Vowel"
end
